Engine/Makefile-objs

```diff
@@ -70,3 +70,43 @@ APEG = libsrc/apeg-1.2.1/adisplay.c libsrc/apeg-1.2.1/getbits.c libsrc/apeg-1.2.
 AASTR = ../Common/libsrc/aastr-0.1.1/aarot.c ../Common/libsrc/aastr-0.1.1/aastr.c ../Common/libsrc/aastr-0.1.1/aautil.c
 
 AL_MIDI_PATCH = libsrc/allegro-4.2.2-agspatch/midi.c
+
+ALDUMB = libsrc/dumb-0.9.2/allegro/alplay.c \
```

Comment:
why is this needed now ?

Reply:
i just checked my build recipe for ags and it was previously possible to use a system-provided libdumb installation. adding this here would force the usage of the sources shipped with ags, afaict.

Reply:
Ahah, my mistake, will remove.

Reply:
I made it optional (on by default in macos). Since the Windows build uses the builtin source, I want to do the same to pick up on any issues.